
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>7BF4 mutated.fasta | Report</title>
    <meta name="description" content="SWISS-MODEL modelling results for 7BF4 mutated.fasta." />
    <link rel="stylesheet" type="text/css" href="js/jquery.fancybox.css"/>
    <style>
			.sequence { font-size: 12px; font-family:"Courier New", Courier, monospace;}
			h2 {font-size: 1.2em; font-family:Arial; margin:20 0 0 0}
			h3 {font-size: 1.0em; font-family:Arial; margin:0 0 0 0}
			
			.borderedTbl>tbody>tr>td, .borderedTbl th{ 
        text-align:center;
  			border-top:1px solid #DDD;
  			border-left:1px solid #DDD;
      }
      .borderedTbl{
        border-bottom:1px solid #DDD;
        border-right:1px solid #DDD; 
      }
      td { 
        -webkit-hyphens: auto;
           -moz-hyphens: auto;
                hyphens: auto; 
              }

			body {
			width:100%; 
			height:100%; 
			background-color: #E7E8EA;
			margin: 0;
			padding: 0;
			word-wrap: break-word;
			font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
      font-size: 14px;

			}
			#content{
			padding:20px;
			width:780px;
			margin-left:auto;
			margin-right:auto;
			background-color:#FFF;}
			
			#sib_footer {
			background-color: #333333;
			color: #F7F8FA;
			font-family: Verdana;
			font-size: 10px;
			font-weight: normal;
			height: 20px;
			letter-spacing: 1px;
			
			width:760px;
      margin-left:auto;
      margin-right:auto;
      padding:0px 20px 0px 20px
			}
			
			#sib_footer a {
  			color: #FFFFFF;
  			font-family: Verdana;
  			font-size: 10px;
  			font-weight: normal;
  			letter-spacing: 1px;
			}
			a, a:link, a:visited {
			color: #D6161D;
			text-decoration: none;
			} 	

			.alTbl{ font-family:'Courier New', Courier, monospace; }
	   img{ border:0; }
			
    .refLinks {
      display: flex;
      flex-wrap: wrap;
      gap: 0px 8px;
    }
    .references li {
      margin: 12px 0px;
    }
    .references .title {
      font-style: italic;
    }
    .references img {
      max-height: 1.2em;
    }
    .references.subtle img {
      -webkit-filter: grayscale(100%);
      filter: grayscale(100%);
    }
    .references a {
      color: inherit;
      display: flex;
      flex-wrap: nowrap;
      align-items: center;
      gap: 2px;
    }
    .references.inline {
      display: flex;
      flex-wrap: wrap;
      gap: 0px 2px;
    }   
	@media print{
	
    #content{
        padding:20px;
        background-color:#FFF;
      }
    body{ 
        width:100%;
        height:100%; 
        background-color: #FFF;
        padding: 0;
      }	
    #sib_footer{
      display:none;
    }
	}
      
    </style>
  </head>
  <body>
  
  <div id="content"> 
    <div style="display: flex; font-size:1.9em; align-items: center; gap: 64px;">
      <img style="height:1.9em" src="images/SWISS-MODEL.tsb.svg"> 
      <div>Homology Modelling Report</div>
    </div>
 
  <h2>Model Building Report</h2>
  <p>
This document lists the results for the homology modelling project &quot;7BF4 mutated.fasta&quot; submitted to SWISS-MODEL workspace
on July 19, 2025, 12:42 p.m..The submitted primary amino acid sequence is given in Table T1.</p>
<p>If you use any results in your research, please cite the relevant publications:</p>
<ul class="references">

  <li><div>Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R, Heer FT, de Beer TAP, Rempfer C, Bordoli L, Lepore R, Schwede T </div><div class="title">SWISS-MODEL: homology modelling of protein structures and complexes. </div><div class="refLinks">Nucleic Acids Res 46, W296-W303. (2018) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/29788355"><img alt="PubMed logo" src="/images/pubmed.png" />29788355</a><a title="Show DOI record" href="https://doi.org/10.1093/nar/gky427"><img alt="DOI logo" src="/images/doi.png" />10.1093/nar/gky427</a> </div></li>
  <li><div>Bienert S, Waterhouse A, de Beer TAP, Tauriello G, Studer G, Bordoli L, Schwede T </div><div class="title">The SWISS-MODEL Repository - new features and functionality. </div><div class="refLinks">Nucleic Acids Res 45, D313-D319. (2017) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/27899672"><img alt="PubMed logo" src="/images/pubmed.png" />27899672</a><a title="Show DOI record" href="https://doi.org/10.1093/nar/gkw1132"><img alt="DOI logo" src="/images/doi.png" />10.1093/nar/gkw1132</a> </div></li>
  <li><div>Studer G, Tauriello G, Bienert S, Biasini M, Johner N, Schwede T </div><div class="title">ProMod3 - A versatile homology modelling toolbox. </div><div class="refLinks">PLOS Comp Biol 17(1), e1008667. (2021) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/33507980"><img alt="PubMed logo" src="/images/pubmed.png" />33507980</a><a title="Show DOI record" href="https://doi.org/10.1371/journal.pcbi.1008667"><img alt="DOI logo" src="/images/doi.png" />10.1371/journal.pcbi.1008667</a> </div></li>
  <li><div>Studer G, Rempfer C, Waterhouse AM, Gumienny R, Haas J, Schwede T </div><div class="title">QMEANDisCo - distance constraints applied on model quality estimation. </div><div class="refLinks">Bioinformatics 36, 1765-1771. (2020) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/31697312"><img alt="PubMed logo" src="/images/pubmed.png" />31697312</a><a title="Show DOI record" href="https://doi.org/10.1093/bioinformatics/btz828"><img alt="DOI logo" src="/images/doi.png" />10.1093/bioinformatics/btz828</a> </div></li>
  <li><div>Bertoni M, Kiefer F, Biasini M, Bordoli L, Schwede T </div><div class="title">Modeling protein quaternary structure of homo- and hetero-oligomers beyond binary interactions by homology. </div><div class="refLinks">Scientific Reports 7. (2017) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/28874689"><img alt="PubMed logo" src="/images/pubmed.png" />28874689</a><a title="Show DOI record" href="https://doi.org/10.1038/s41598-017-09654-8"><img alt="DOI logo" src="/images/doi.png" />10.1038/s41598-017-09654-8</a> </div></li>

</ul>

<h2>Results</h2>
<p>
The SWISS-MODEL template library (SMTL version 2025-07-16, PDB release 2025-07-11) was searched with 
BLAST (<a href="#blastp">Camacho et al.</a>) and HHblits (<a href="#hhblits">Steinegger et al.</a>) 

 for evolutionary related structures matching the target sequence in Table T1. For details on the template search, see Materials and Methods. Overall 267 templates were found (Table T2).</p>


<h2>Models</h2>


<p>The following model was built (see Materials and Methods "Model Building"):</p>

      

      <table id="model_01" class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5>
  		<thead>
        <tr>
        <th><h3>Model #01</h3></th>
        <th>File</th>
        <th>Built with</th>
        <th>Oligo-State</th>
        <th>Ligands</th>
        <th title="Global Model Quality Estimate">GMQE</th>
        <th>QMEANDisCo Global</th>
        </tr>
      </thead>
      <tbody>
      <tr>
        <td><a href="model/01/01.png" target="_blank"><img height="60" src="model/01/01.png"></a>
        </td>
  	    <td><a href="model/01/model.pdb" target="_blank">PDB</a></td>
  		<td>ProMod3 3.5.0</td>
        <td>monomer </td>
        <td>
          <div style="max-width:350px;"> 1 x APR: ADENOSINE-5-DIPHOSPHORIBOSE;
          </div>
        </td>
        <td>0.96</td>
        <td><span id="avgLocalScore_01">0.90</span> &plusmn; 0.07</td>
      </tr>
     </table>
  	
  	<table width="100%" cellspacing=5>
  	<tr>
  	    <td width="30%">
       	 <table id="qmeanGlobalTbl01"></table>
        </td>
          <td width="33%">
            <a href="#lq-chart-01" rel="lq-01" class="fancybox">
              <div id="lq-chart-01-thumb" style="margin:12px 4px;"></div>
            </a>
            <div id="lq-chart-01" style="display:none"></div>
          </td>

          <td width="33%">
            <a href="#qn-chart-01" class="fancybox">
              <div id="qn-chart-01-thumb" style="margin:12px 4px;"></div>
            </a>
            <div id="qn-chart-01" style="display:none"></div>
          </td>
  	 </tr>
  	</table>


  	<table class="borderedTbl">
  		<thead>
  			<tr>
  				<th>Template</th>
  				<th>Seq Identity</th>
  				<th>Oligo-state</th>
  				<th>QSQE</th>
  				<th>Found by</th>
  				<th>Method</th>
  				<th>Resolution</th>
  				<th>Seq Similarity</th>
  				<th>Range</th>
  				<th>Coverage</th>
  				<th>Description</th>
  			</tr>
  		</thead>
  		<tbody>
        
        
        <tr>
  				<td>
                
                  <a href="model/01/templates/6z5t.1.A.pdb.gz" target="_blank">6z5t.1.A</a>
                
            
          </td>
  				<td>98.83</td>
  				<td>monomer</td>
  				<td>0.00</td>
  				<td>BLAST</td>
  				<td>X-ray</td>
  				<td>1.57&Aring;</td>
  				<td>0.60</td>
  				<td>3 - 173

        </td>
  				<td>0.99</td>
  				<td>
          Replicase polyprotein 1ab</td>
  			</tr>
  		</tbody>
  	</table> 
  	
    

    
    <br>
    <h3>Included Ligands</h3>
  	<table width="100%" class="borderedTbl">
  		<thead>
  			<tr>
	  			<th>Ligand</th>
	  			<th>Description</th>
	  		</tr>
	  	</thead>
		<tbody>
	  
      <tr><td>1 x APR</td>
      <td><div style="max-width:650px;">ADENOSINE-5-DIPHOSPHORIBOSE</div></td>
      </tr>
    
    </tbody>
    </table>

  
    <br>
    <h3>Excluded ligands</h3>
    <table width="100%" class="borderedTbl">
      <thead>
        <tr>
          <th>Ligand Name.Number</th>
          <th>Reason for Exclusion</th>
          <th>Description</th>
        </tr>
      </thead>
    <tbody>
    
      
      <tr><td>NA.2</td><td>
      Not biologically relevant.
      </td>
      <td><div style="max-width:650px;">SODIUM ION</div></td>
      </tr>
      
    
		</tbody>
	</table>
  

<br>


<pre>Target    GHMVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAAALNKATNNAMQVESDDYIATNGPLKVGG<br>6z5t.1.A  --EVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAGALNKATNNAMQVESDDYIATNGPLKVGG<br><br>Target    SCVLSGHNLAKHCLHVVGPNVNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSLRVCVDTVRTNVYLAVFDKN<br>6z5t.1.A  SCVLSGHNLAKHCLHVVGPNVNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSLRVCVDTVRTNVYLAVFDKN<br><br>Target    LYDKLVSSFLEMK<br>6z5t.1.A  LYDKLVSSFLEMK<br><br></pre>


  <br>
  <hr><br>
  





<h2>Materials and Methods</h2>



<h2>Template Search</h2>
<p>Template search with BLAST and HHblits
 has been performed against the SWISS-MODEL template library (SMTL, last update: 2025-07-16, last included PDB release: 2025-07-11).</p>

<p>
The target sequence was searched with BLAST against the primary amino acid sequence contained in the SMTL.


A total of 76 templates were found.


</p> 


<p>
An initial HHblits profile has been built using the procedure outlined in (<a href="#hhblits">Steinegger et al.</a>), followed by 1 iteration of HHblits against Uniclust30 (<a href="#uniclust">Mirdita, von den Driesch et al.</a>). The obtained profile has then be searched against all profiles of the SMTL. A total of 204 templates were found.</p>
















<h2>Model Building</h2>
<p>
Models are built based on the target-template alignment using ProMod3 (<a href="#promod3">Studer et al.</a>). Coordinates which are conserved between the target and the template are copied from the template to the model. Insertions and deletions are remodelled using a fragment library. Side chains are then rebuilt. Finally, the geometry of the resulting model is regularized by using a force field.
</p>

<h2>Model Quality Estimation</h2>
<p>
The global and per-residue model quality has been assessed using the QMEAN scoring function (<a href="#qmean">Studer et al.</a>).
</p>

<h2>Ligand Modelling</h2>
<p>
Ligands present in the template structure are transferred by homology to the model when the following criteria are met: (a) The ligands are annotated as biologically relevant in the template library, (b) the ligand is in contact with the model, (c) the ligand is not clashing with the protein, (d) the residues in contact with the ligand are conserved between the target and the template. If any of these four criteria is not satisfied, a certain ligand will not be included in the model. The model summary includes information on why and which ligand has not been included.</p>

<h2>Oligomeric State Conservation</h2>
<p>
The quaternary structure annotation of the template is used to model the target sequence in its oligomeric form. The method (<a href="#qsqe">Bertoni et al.</a>) is based on a supervised machine learning algorithm, Support Vector Machines (SVM), which combines interface conservation, structural clustering, and other template features to provide a quaternary structure quality estimate (QSQE). The QSQE score is a number between 0 and 1, reflecting the expected accuracy of the interchain contacts for a model built based a given alignment and template. Higher numbers indicate higher reliability. This complements the GMQE score which estimates the accuracy of the tertiary structure of the resulting model.</p>

<h2>References</h2>
<p>
  <ul class="references">
    
      <li><div>Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL </div><div class="title">BLAST+: architecture and applications. </div><div class="refLinks">BMC Bioinformatics, 10, 421-430. (2009) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/20003500"><img alt="PubMed logo" src="/images/pubmed.png" />20003500</a><a title="Show DOI record" href="https://doi.org/10.1186/1471-2105-10-421"><img alt="DOI logo" src="/images/doi.png" />10.1186/1471-2105-10-421</a> </div></li>
      <li><div>Steinegger M, Meier M, Mirdita M, Vöhringer H, Haunsberger SJ, Söding J </div><div class="title">HH-suite3 for fast remote homology detection and deep protein annotation. </div><div class="refLinks">BMC Bioinformatics 20, 473. (2019) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/31521110"><img alt="PubMed logo" src="/images/pubmed.png" />31521110</a><a title="Show DOI record" href="https://doi.org/10.1186/s12859-019-3019-7"><img alt="DOI logo" src="/images/doi.png" />10.1186/s12859-019-3019-7</a> </div></li>
      
        <li><div>Mirdita M, von den Driesch L, Galiez C, Martin MJ, Söding J, Steinegger M </div><div class="title">Uniclust databases of clustered and deeply annotated protein sequences and alignments. </div><div class="refLinks">Nucleic Acids Res, 45, D170–D176. (2016) <a title="Show PubMed record" href="https://pubmed.ncbi.nlm.nih.gov/27899574"><img alt="PubMed logo" src="/images/pubmed.png" />27899574</a><a title="Show DOI record" href="https://doi.org/10.1093/nar/gkw1081"><img alt="DOI logo" src="/images/doi.png" />10.1093/nar/gkw1081</a> </div></li>
      
    
  </ul>

<h2>Table T1:</h2>
<p>Primary amino acid sequence for which templates were searched and models were built.</p>
<div class="sequence">GHMVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAAALNKATNNAMQVESDDYIATNGPLKVGGSCVLSGHNLAKHCLHVVGPN<br>VNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSLRVCVDTVRTNVYLAVFDKNLYDKLVSSFLEMK<br></div>


<h2>Table T2:</h2>
<table class="borderedTbl">
<thead>
	<tr>
		<th>Template</th>
		<th>Seq Identity</th>
		<th>Oligo-state</th>
		<th>QSQE</th>
		<th>Found by</th>
		<th>Method</th>
		<th>Resolution</th>
		<th>Seq Similarity</th>
		<th>Coverage</th>
		<th>Description</th>
	</tr>
</thead>
<tbody>

<tr>
		<td>6z5t.1.A</td>
		<td>98.83</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.57&Aring;</td>
		<td>0.60</td>
		<td>0.99</td>
		<td>Replicase polyprotein 1ab</td>
	</tr>
<tr>
		<td>6woj.1.A</td>
		<td>98.26</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>2.20&Aring;</td>
		<td>0.60</td>
		<td>0.99</td>
		<td>Non-structural protein 3</td>
	</tr>
<tr>
		<td>6wey.1.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>0.95&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>Non-structural protein 3</td>
	</tr>
<tr>
		<td>6ywl.2.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>HHblits</td>
		<td>X-ray</td>
		<td>2.50&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>NSP3 macrodomain</td>
	</tr>
<tr>
		<td>6ywk.3.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>HHblits</td>
		<td>X-ray</td>
		<td>2.20&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>NSP3 macrodomain</td>
	</tr>
<tr>
		<td>6ywm.2.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>HHblits</td>
		<td>X-ray</td>
		<td>2.16&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>NSP3 macrodomain</td>
	</tr>
<tr>
		<td>7bf6.1.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>HHblits</td>
		<td>X-ray</td>
		<td>2.15&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>Papain-like protease nsp3</td>
	</tr>
<tr>
		<td>7c33.1.A</td>
		<td>99.41</td>
		<td>monomer</td>
		<td>-</td>
		<td>HHblits</td>
		<td>X-ray</td>
		<td>3.83&Aring;</td>
		<td>0.61</td>
		<td>0.98</td>
		<td>Non-structural protein 3</td>
	</tr>
<tr>
		<td>2acf.2.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.40&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Replicase polyprotein 1ab</td>
	</tr>
<tr>
		<td>8cb3.2.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.57&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Papain-like protease nsp3</td>
	</tr>
<tr>
		<td>8cb3.3.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.57&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Papain-like protease nsp3</td>
	</tr>
<tr>
		<td>2fav.2.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.80&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Replicase polyprotein 1ab (pp1ab) (ORF1AB)</td>
	</tr>
<tr>
		<td>8cb3.1.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.57&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Papain-like protease nsp3</td>
	</tr>
<tr>
		<td>2fav.3.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.80&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Replicase polyprotein 1ab (pp1ab) (ORF1AB)</td>
	</tr>
<tr>
		<td>2acf.1.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.40&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Replicase polyprotein 1ab</td>
	</tr>
<tr>
		<td>2fav.1.A</td>
		<td>73.21</td>
		<td>monomer</td>
		<td>-</td>
		<td>BLAST</td>
		<td>X-ray</td>
		<td>1.80&Aring;</td>
		<td>0.52</td>
		<td>0.97</td>
		<td>Replicase polyprotein 1ab (pp1ab) (ORF1AB)</td>
	</tr>
</tbody>
</table>


<br>The table above shows the top 16 filtered templates. A further 189 templates were found which were considered to be less suitable for modelling than the filtered list.<br>
1hjz.1.A, 1spv.1.A, 1ty8.2.A, 1vhu.1.A, 1yd9.1.A, 1yd9.3.A, 1yd9.4.A, 1zr3.2.A, 1zr3.3.A, 1zr5.1.A, 2acf.1.A, 2acf.2.A, 2acf.4.A, 2afc.1.A, 2bfq.1.A, 2dx6.1.A, 2eee.1.A, 2fav.1.A, 2fav.2.A, 2fav.3.A, 2fg1.1.A, 2fxk.2.A, 2l8r.1.A, 2lgr.1.A, 2vri.1.A, 2x47.1.A, 2xd7.1.A, 3ejf.1.A, 3ejg.1.A, 3eke.1.A, 3eti.1.A, 3ew5.1.A, 3ew5.2.A, 3ewo.1.A, 3ewo.1.B, 3ewq.1.A, 3gpg.2.A, 3gpq.3.A, 3gpq.4.A, 3gqe.1.A, 3gqe.2.A, 3gqo.3.A, 3gqo.4.A, 3iid.1.A, 3iif.1.A, 3iif.2.A, 3iif.3.A, 3jzt.4.A, 3q6z.1.A, 3q71.1.A, 3sig.1.A, 3sij.1.A, 3v2b.1.A, 3v45.1.A, 3vfq.1.A, 4abl.1.A, 4ess.1.A, 4etj.1.A, 4etk.1.A, 4etk.2.A, 4gua.1.A, 4gvv.1.A, 4gvv.2.A, 4gvv.3.A, 4gvw.1.A, 4iqy.1.A, 4j4z.1.A, 4j5q.1.A, 4j5r.2.A, 4j5s.4.A, 4jca.1.A, 4jgk.1.A, 4jgk.3.A, 4jll.1.A, 4jvv.1.A, 4k0c.1.A, 4k0c.2.A, 4kyb.1.A, 4kyb.2.A, 4uml.1.A, 5ail.1.A, 5cb3.1.A, 5cb5.1.A, 5cms.1.A, 5dus.1.A, 5e3b.1.A, 5fsu.1.A, 5fsu.2.A, 5fsz.1.A, 5fud.1.A, 5hih.1.A, 5iit.1.A, 5iq5.1.A, 5isn.1.A, 5kiv.1.A, 5l9k.1.A, 5lau.1.A, 5lbp.1.A, 5lcc.1.A, 5lcc.2.A, 5lnc.1.A, 5lnc.2.A, 5lw0.1.A, 5lw6.1.A, 5m31.1.A, 5m3e.1.A, 5m3i.1.A, 5mqx.1.A, 5o2d.1.A, 5rts.2.A, 5s4c.1.A, 5zdg.1.A, 5zu7.1.A, 6fx7.1.A, 6fy5.1.A, 6fy5.2.A, 6lfq.1.A, 6lfr.1.A, 6lfs.1.A, 6lft.1.A, 6lfu.1.A, 6lfu.2.A, 6lh4.1.A, 6lh4.4.A, 6ln0.1.A, 6mea.1.A, 6r0g.2.A, 6vxs.2.A, 6w02.1.A, 6w02.2.A, 6w0t.1.D, 6wey.1.A, 6woj.1.A, 6y4y.1.A, 6y4z.1.A, 6z5t.1.A, 7d2c.1.A, 7d3y.1.A, 7d3y.1.B, 7epu.1.B, 7h0s.2.A, 7h6j.1.A, 7h78.2.A, 7h7f.3.A, 7h7x.4.A, 7jme.1.A, 7kg3.1.A, 7kqw.1.A, 7kr0.1.A, 7kr1.1.A, 7ny6.1.A, 7ny7.1.A, 7omu.1.A, 7p27.1.A, 7twt.1.A, 7tx3.2.A, 7tx4.1.A, 7tx5.1.A, 7wr6.1.B, 8azc.1.B, 8azd.1.A, 8cb3.1.A, 8cb3.2.A, 8cb3.3.A, 8gia.1.A, 8gia.2.A, 8p0c.1.A, 8p0c.2.A, 8rsi.1.A, 8rsj.1.A, 8rsk.2.A, 8rsl.1.A, 8rsm.1.A, 8rsn.1.A, 8sh6.1.A, 8sh6.2.A, 8sh8.1.A, 8tv6.1.A, 8tv7.1.A, 9fia.1.Z, 9gub.2.A, 9hqv.44.A, 9i05.48.A, 9qyd.1.A, 9qye.1.A, 9qyf.1.A, 9qyg.1.A, 9qyg.1.B, 9qyh.2.A
 


	</div>
	
  <div id="sib_footer">			
   <div id="sib_footer_content"> 
       <a href="https://www.sib.swiss">Swiss Institute of Bioinformatics</a>
       <a href="mailto:help-swissmodel@unibas.ch" style="margin-left:16px;">Contact Us</a>
      <div id="sib_footer_right" style="float:right"></div>
  	</div>
	</div>
  
  <script src="js/jquery-3.7.0.min.js"></script>
  <script src="js/jquery.fancybox.pack.js"></script> 
  <script src="js/raphael-min.js"></script>
  <script src="js/smng.min.js"></script>
  <script>
 var localQMEANNormImage='images/qmean_norm.png';
 var modelData = {
   
      "01":{
        'qmean_local' : { 
        "A":[null,null,0.63,0.75,0.86,0.89,0.92,0.93,0.95,0.92,0.92,0.9,0.91,0.9,0.9,0.91,0.95,0.94,0.94,0.91,0.93,0.96,0.93,0.94,0.93,0.92,0.91,0.95,0.9,0.9,0.91,0.85,0.93,0.94,0.96,0.97,0.98,0.94,0.96,0.94,0.89,0.9,0.89,0.9,0.88,0.87,0.81,0.87,0.91,0.93,0.94,0.94,0.94,0.92,0.92,0.88,0.95,0.88,0.86,0.86,0.92,0.88,0.85,0.86,0.88,0.88,0.87,0.86,0.87,0.86,0.84,0.83,0.85,0.9,0.92,0.93,0.9,0.96,0.98,0.98,0.96,0.96,0.92,0.92,0.91,0.92,0.89,0.9,0.91,0.93,0.89,0.91,0.96,0.95,0.94,0.97,0.95,0.96,0.94,0.9,0.92,0.86,0.84,0.88,0.87,0.92,0.92,0.87,0.92,0.93,0.91,0.94,0.97,0.95,0.92,0.94,0.94,0.93,0.87,0.89,0.88,0.93,0.94,0.95,0.99,0.96,0.94,0.94,0.93,0.93,0.89,0.85,0.91,0.93,0.95,0.91,0.92,0.89,0.88,0.94,0.93,0.85,0.94,0.96,0.93,0.92,0.95,0.96,0.87,0.89,0.94,0.97,0.95,0.95,0.97,0.95,0.89,0.89,0.86,0.83,0.86,0.89,0.86,0.82,0.87,0.87,0.83,0.8,0.79,0.74,0.75,0.68,0.77] },
        'qmean_global':{"qmean4_norm_score":0.7724611673418611,"qmean4_z_score":-0.19425321808941529,"qmean6_norm_score":0.761392451862201,"qmean6_z_score":-0.4301949565797545,"interaction_norm_score":-0.028242869588264947,"interaction_z_score":0.5002135853617635,"cbeta_norm_score":-0.01459849935786995,"cbeta_z_score":0.18660562679583906,"packing_norm_score":-0.3762059105971722,"packing_z_score":0.0950619192230818,"torsion_norm_score":-0.3002714744084487,"torsion_z_score":-0.30285166796844637,"ss_agreement_norm_score":0.6670021917126332,"ss_agreement_z_score":0.01961412066624584,"acc_agreement_norm_score":0.5964912280701754,"acc_agreement_z_score":-0.69550848140106,"avg_local_score":0.9043732619690799,"avg_local_score_error":0.066}
      }
    
 }  


 $(document).ready( function()
   {  
      drawQMEAN('01')
   }) ;

function drawQMEAN(id)
{
  var resCount = 0;
  var local_scores = [];
  for(chain in modelData[id].qmean_local)
  {
    var ls = {'chain':chain,'scores':[]};

    for(var i=0; i<modelData[id].qmean_local[chain].length; i++)
    {
        ls['scores'].push([i, modelData[id].qmean_local[chain][i]]);
        if(modelData[id].qmean_local[chain][i])
          resCount++;
    }
    local_scores.push(ls);
  }

  $('#avgLocalScore_'+id)
    .css({'background-color':
          qmeanLocalColour(modelData[id].qmean_global.avg_local_score, .9),
          'padding':'2px 8px 2px 8px',
          'color':'white',
          'border-radius':'1em',               
          'display':'inline-block'});

  drawQMEANLocal(local_scores, modelData[id].qmean_global.avg_local_score, id);

  drawQMEANGlobal(modelData[id]['qmean_global'], resCount, id, false);

  $('a[rel=lq-'+id+'],a[href="#qn-chart-'+id+'"]')
    .fancybox(
      {
        padding:30, 
        'cyclic':true,
        'hideOnContentClick':true,
        beforeShow:function(a,b)
        {
          var id = $(this.content[0]).first().attr('id');
          if(id.indexOf('lq-chart')>-1)
            $('#'+id).closest('.fancybox-outer').after(
               '<a style="color:#666;float:right;" onclick="svgToPNG(\''+id+'\',\'Local_quality_estimate.png\');">&#128247; Save as PNG</a>'
               );
          else if(id.indexOf('qn-chart')>-1)
            $('#'+id).after(
               '<a style="color:#666;float:right;" onclick="svgToPNG(\''+id+'\',\'Quality_comparison.png\');">&#128247; Save as PNG</a>'
               );
        }
      });
}




  </script>

  </body>
</html>
